src/project/internal/projectuiactions.cpp

```diff
@@ -79,28 +79,27 @@ const UiActionList ProjectUiActions::m_actions = {
              mu::context::UiCtxAny,
              mu::context::CTX_ANY,
              TranslatableString("action", "Save to clo&ud…"),
-             TranslatableString("action", "Save to cloud…"),
+             TranslatableString("action", "Save this score to your MuseScore.com account"),
```

Comment:
Note that this string also appears, without context, in the list in Preferences > Shortcuts

(yes, this problem keeps coming up, we _really_ need to decouple tooltips from Preferences > Shortcuts; previous time was https://github.com/musescore/MuseScore/pull/29005#discussion_r2243308475)

Reply:
Good solution, for now. 
We could of course use this in all individual places where we want tooltips... that's perhaps even better than trying to specify those application-wide via UiAction. What do you think, should we do that, or should we still plan to add a new field to UiAction?

Reply:
Actually, I don’t have a definitive answer. Overall, I like the idea of adding a general description in UiActions, and if there are unique places where the description needs slight adjustments, we handle that in the models. But I’m afraid there might be many such cases.